### DISCUSSION TITLE
Documentatie op Github Pages

### DISCUSSION THREAD

**Discussion by ElisabethdL** [mvwestendorp/v-en-a#80] [Documentatie]
We moeten een publieke website hebben met duidelijke uitleg.
Voorstel:
- tekst en screenshots overnemen uit leermodule KUL
- screencasts maken (max. 30 seconden)
- vervangen of toevoegen naar gelang ze af zijn

Te beslissen:
- welke screencasts --> eerst installatie programma
- wie pakt deze aan
- deadline?

**Comment by ShanaLudikhuyze** [comment]
hoe beheren we de video's? via youtube of google drive? generisch account voor v&a nodig? paswoordbeheer?

**Comment by ElisabethdL** [comment]
Opmerkingen:
- filmpjes op YouTube zetten of Google Drive? wiens account?
- account van de bib rechten? contactpersoon duidelijk!

Voorstel voor screencast: mini-scenario's van een paar zinnen.

**Comment by ElisabethdL** [comment]
Starten met handleiding op Github:
- https://drive.google.com/drive/folders/10aiQiLG7X_BB0Q7VuaQ6u8AWdLJbMJ2j?usp=sharing
- https://v-a-collaboration.github.io/v-en-a/pages/handleiding.html

1 pagina per leermodule, dus niet per pagina in de leermodule.

Snelle markdown:
- https://daringfireball.net/projects/markdown/
- https://github.github.com/gfm/
- https://www.markdownguide.org/getting-started/

**Comment by ElisabethdL** [comment]
Uitzoeken hoe we binnen de website een interne structuur maken (mapje "handleiding" in pages).
Van één module naar een andere met pijltjes? Apart menu aan linkerzijde?

Per topic binnen leermodule: interne links maken, bovenaan pagina menuutje zetten.
